imports/plugins/included/default-theme/client/styles/variables.less

```diff
@@ -486,6 +486,7 @@
 @zindex-navbar-fixed:      1030;
 @zindex-modal-background:  1040;
 @zindex-modal:             1050;
+@zindex-select:            9999;
```

Comment:
So this gives us no room to ever have anything higher. Is that really what we want to do?